Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py

```diff
@@ -0,0 +1,511 @@
+) -> Dict[str, dict]:
+    """Parse group data as received from CrowdStrike FalconX API into Demisto's conventions
+    the output from the API is a dict that contains the keys: meta, resources and errors
+    the meta contains a "quota" dict
+    the "resources" is an array that contains the sandbox dict
+    the function filters the wanted params from the api result
+    :param api_res: the api result from the http request
+    :param meta_fields: the wanted params that appear in the mate section
+    :param quota_fields: the wanted params that appear in the quota section
+    :param resources_fields: the wanted params that appear in the resources section
+    :param sandbox_filds: the wanted params that appear in the sandbox section
+    :return: a dict based on api_res with the wanted params only
+    """
+    if api_res.get("errors"):
+        # if there is an error in the api result, return only the error
+        return api_res.get("errors")
+
+    api_res_meta, api_res_quota, api_res_resources, api_res_sandbox = {}, {}, {}, {}
+    resources_group_outputs, sandbox_group_outputs = {}, {}
+
+    api_res_meta = api_res.get("meta")
+    if api_res_meta:
+        api_res_quota = api_res_meta.get("quota")
+
+    meta_group_outputs = add_outputs_from_dict(api_res_meta, meta_fields)
+    quota_group_outputs = add_outputs_from_dict(api_res_quota, quota_fields)
+
+    if api_res.get("resources"):
+        # depended on the command, the resources section can be a str list or a list that contains
+        # only one argument which is a dict
+        if type(api_res.get("resources")[0]) == dict:
+            api_res_resources = api_res.get("resources")[0]
+            resources_group_outputs = add_outputs_from_dict(api_res_resources, resources_fields)
+
+            if api_res_resources and api_res_resources.get("sandbox"):
+                api_res_sandbox = api_res_resources.get("sandbox")[0]
+                sandbox_group_outputs = add_outputs_from_dict(api_res_sandbox, sandbox_filds)
+        else:
+            # the resources section is a list of strings
+            resources_group_outputs = {"resources": api_res.get("resources")}
+
+    merged_dicts = {**meta_group_outputs, **quota_group_outputs, **resources_group_outputs, **sandbox_group_outputs}
+
+    return {f'csfalconx.resource(val.resource === obj.resource)': merged_dicts}
+
+
+def test_module(client):
+    """
+    If a client was made then an accesses token was successfully reached,
+    therefor the username and password are valid and a connection was made
+    additionally, checks if not using all the optional quota
+    :param client: the client object with an access token
+    :return: ok if got a valid accesses token and not all the quota is used at the moment
+    """
+    _, _, output = check_quota_status_command(client)
+    total = output.get("meta").get("quota").get("total")
+    used = output.get("meta").get("quota").get("used")
+    if total == used:
```

Comment:
Just to be on the safe side
```suggestion
    if total => used:
```